packages/web/src/features/languages/ImportLanguageGlossesView.tsx

```diff
@@ -46,6 +47,8 @@ export default function ImportLanguageGlossesView() {
   const { language, importLanguages, currentJob } = useLoaderData() as Awaited<
     ReturnType<typeof importLanguageGlossesLoader>
   >;
+  useTitle(`Import ${language.data.name} Glosses`);
```

Comment:
@arrocke you didn't mention this at all in the issue description, but we'll want the tab title to be shown in the currently select language, right?

Reply:
Yep good call. Maybe we should have a nested object in the common json file for all of the page titles.